packages/terra-overlay/src/Overlay.jsx

```diff
@@ -81,9 +82,10 @@ class Overlay extends React.Component {
       this.container = node.parentNode;
       this.containerChildren = this.container.children;
       this.disableContainerChildrenFocus();
+      this.containerChildren[0].focus();
     } else {
       document.documentElement.style.overflow = 'hidden';
-      this.container = null;
+      document.documentElement.getElementsByClassName('base')[0].setAttribute('aria-hidden', 'true'); // prevent screen reader from moving to hidden content
```

Comment:
Let's update this to:
```suggestion
      document.querySelector(selector).setAttribute('aria-hidden', 'true'); // prevent screen reader from moving to hidden content
```

Where `selector` is a variable based on a new overlay prop, `rootSelector`.

```jsx
const selector = this.props.rootSelector;
```

We'll default this prop to `data-terra-base`, but this allows users to pass in their own "root selector" if they are not using terra-base.